media-api/app/lib/MediaApiConfig.scala

```diff
@@ -87,7 +87,7 @@ class MediaApiConfig(override val configuration: Configuration) extends CommonCo
 
   lazy val persistedRootCollections: List[String] = properties.get("persistence.collections") match {
     case Some(collections) => collections.split(',').toList
-    case None => List("GNM Archive")
+    case None => List("GNM Archive", "Daily Edit")
```

Comment:
We're reading persisted collections from config about (line 88). This line is the defaults should config be empty. I think it would be better to add this new collection to configuration.

Reply:
Happy to do this, but couldn’t find where do we store the config…